docs/content/doc/development/oauth2-provider.en-us.md

```diff
@@ -44,42 +44,53 @@ To use the Authorization Code Grant as a third party application it is required
 
 ## Scopes
 
-Gitea supports the following scopes for tokens:
+Gitea supports scoped access tokens, which allow users the ability to restrict tokens to operate only on selected url routes. Scopes are grouped by high-level API routes, and further refined to the following:
+
+- `read`: `GET` routes
+- `write`: `POST`, `PUT`, and `PATCH` routes (in addition to `GET`)
+- `delete`: `DELETE` routes (in addition to `POST`, `PUT`, `PATCH` and `GET`)
+
+Gitea token scopes are as follows:
```

Comment:
The "public only" is not mentioned here but appears in code, I still have difficulty to understand what is "public only" or what it affects .....